test/results/generate_02.ics

```diff
@@ -10,7 +10,7 @@ DTEND:20131004T231500Z
 SUMMARY:Sample Event
 LOCATION:localhost
 DESCRIPTION:Lorem ipsum dolor sit amet\, consetetur sadipscing elitr\, sed
-  diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam era
- t\, sed diam voluptua.\nbeep boop
+ diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam era
+  t\, sed diam voluptua.\nbeep boop

```

Comment:
@ocelotsloth 
However it still failed.
I check the RFC 5545 3.1 again and found that it DOESN'T need to add more and more spaces for each line, that exta space comes from content itself! 
```
For example, the line:

     DESCRIPTION:This is a long description that exists on a long line.

   Can be represented as:

     DESCRIPTION:This is a lo
      ng description
       that exists on a long line.
```
now I'm trying to reproduce the case that I used to have. I'm sorry for this redundant pr and thank you for your kind help